appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
@gitars 

One way of debugging without having to build apps is to write and run unit tests.  That has the added advantage of adding to the general test suite for everyone to use.  Unfortunately, there aren't any existing such tests for the event handling stuff for you to just add to - you would have to create your own.  But if you're experienced with Java and junit you should be able to write tests for the stuff in EventDispatcher.java.

It's harder for the stuff in runtime.scm, of course, since that requires knowledge of Scheme.  There is some stuff in [YailEvalTest.java](https://github.com/gitars/appinventor-sources/blob/master/appinventor/buildserver/tests/com/google/appinventor/buildserver/YailEvalTest.java) for testing Scheme code using junit that might be worthing looking at.